app/controllers/trainees/qts_recommendations_controller.rb

```diff
@@ -7,13 +7,13 @@ def create
 
       RecommendForQtsJob.perform_later(trainee.id)
 
-      redirect_to recommended_trainee_outcome_details_path(trainee_id: trainee.id)
+      redirect_to recommended_trainee_outcome_details_path(trainee_id: trainee.slug)
```

Comment:
We can also just use `recommended_trainee_outcome_details_path(trainee)`. I think there's more opportunity to remove the explicit `trainee_id` keyword argument in various places and replace it with just a trainee object. No big deal if it's too much work.

Reply:
agreed but cleaned up some of `.slug` calls for now in https://github.com/DFE-Digital/register-trainee-teachers/pull/405/commits/9e311434810f371cafa169c0e7559182064629a3